composer/callbacks/image_monitor.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Monitor train and eval images."""
+from typing import Any, Callable, Tuple, Union
+
+import torch
+
+from composer.core import Callback, State, Time, TimeUnit
+from composer.loggers import Logger
+from composer.utils.import_helpers import MissingConditionalImportError
+
+__all__ = ['ImageMonitor']
+
+
+class ImageMonitor(Callback):
```

Comment:
What are your thoughts on `ImageVisualizer`? It seems more informative here for a user look for a callback. `ImageMonitor` could mean many things, whereas `ImageVisualizer` is a bit more specific.

Reply:
or `ImageLogger`?

Reply:
Agree, `ImageVisualizer` is better, I'll make that change.

Reply:
nice!